ui/tests/integration/components/secret-engine/configure-aws-test.js

```diff
@@ -123,6 +136,24 @@ module('Integration | Component | SecretEngine/ConfigureAws', function (hooks) {
           .isNotChecked('identityTokenTtl is cleared after toggling accessType');
       });
 
+      test('it does not clear global issuer when toggling accessType', async function (assert) {
+        this.issuerConfig = createConfig(this.store, this.id, 'issuer');
+        await this.renderComponent();
+        await click(SES.aws.accessType('wif'));
+        assert
+          .dom(GENERAL.inputByAttr('issuer'))
+          .hasValue(this.issuerConfig.issuer, 'issuer is what is sent in my the model on first load');
+        await fillIn(GENERAL.inputByAttr('issuer'), 'http://ive-changed');
+        await click(SES.aws.accessType('iam'));
+        await click(SES.aws.accessType('wif'));
+        assert
+          .dom(GENERAL.inputByAttr('issuer'))
+          .hasValue(
+            this.issuerConfig.issuer,
```

Comment:
This isn't a very reliable test because we're just checking that the value is what we pass it. A better test is checking that it has the value 'issuer' which is what it was originally

Reply:
A little confusing here, and maybe I can clear this up somehow in the test. But the value passed is the global value; what is on the issuerConfig. The test is changing this value then toggling to IAM values and then toggling back and just making sure that the changed value doesn't stay. So we do want to check what's on the issuerConfig. 

Screenshots:
1. on load. 
![image](https://github.com/user-attachments/assets/5f820fbb-b8b7-4288-8614-7a225301b1c7)
2. fillIn (typing something in to simulate a user thinking about filling in something but then changing their mind).
![image](https://github.com/user-attachments/assets/6aac2aa7-99c0-46eb-9501-d5ccd355cc2c)
3. Toggle to iam and then back and it's still the original passed in value
![image](https://github.com/user-attachments/assets/1ec82d81-e05f-46f2-8df6-8ebd0f048349)